helix-term/src/commands.rs

```diff
@@ -9,11 +9,10 @@ use helix_core::{
     Tendril, Transaction,
 };
 
-use helix_view::{
-    document::{IndentStyle, Mode},
-    view::{View, PADDING},
-    Document, DocumentId, Editor, ViewId,
-};
+use helix_view::document::{IndentStyle, Mode};
```

Comment:
Nit: I don't like the import change here

Reply:
Haha

Reply:
Fixed